.reuse/dep5

```diff
@@ -0,0 +1,52 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: hedgedoc
+Upstream-Contact: The HedgeDoc developers <license@hedgedoc.org>
+Source: https://github.com/hedgedoc/hedgedoc
+
+Files: .idea/**
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC0-1.0
+
+Files: .github/ISSUE_TEMPLATE/*.md
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC-BY-SA-4.0
+
+Files: .github/pull_request_template.md
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC-BY-SA-4.0
+
+Files: docs/**/*.md
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC-BY-SA-4.0
+
+Files: docs/mkdocs.yml
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC0-1.0
+
+Files: docs/requirements.txt
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC0-1.0
+
+Files: docs/**/*.png
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC0-1.0
```

Comment:
Just asking: Images for the docs under CC0? I just thought that it would be - like the other docs content - under CC-BY-SA-4.0.

Reply:
I'll change that

Reply:
done